src/karts/ghost_kart.cpp

```diff
@@ -130,9 +135,45 @@ void GhostKart::update(int ticks)
         m_all_physic_info[idx].m_steer, m_all_physic_info[idx].m_speed,
         /*lean*/0.0f, idx);
 
+    // Attachment
+    // Uses custom format so that replay files remain valid if
+    // internal numbering changes.
+
+    int attach_ticks = 9000;
```

Comment:
Hardcoded arbitrary constant

Reply:
We have in several instances delays in seconds hard coded (e.g. how long it takes for an item to reappear). While it might not be worth to make everything configurable, at least use:
stk_config->time2Ticks(75);   
so that we can change the tick duration without changing the game play.

Reply:
Yeah, that part is arbitrary ; it's just putting a big enough number (this corresponds to over a minute) to make sure the attachment won't go away until it is explicitly cleared. As the replay file knows when it should be done, there is no point in trying to recalculate it, especially as the values used to calculate it may have changed since the replay file has been saved. The attachment can't be set each frame, because this would reset its graphic size and keep it tiny.

The ticks value can't be read from the replay file as it's not saved here (and considering it would only serve a few times per replay at most, saving it in hundreds of lines may be overkill).

The downside is that with this way of doing it, the clocktime of the bomb will be wrong and that the bubbleshield won't flicker before disappearing (don't matter currently as none of it can be gotten in time-trial, but to keep in mind if expanding replays to more type of races).